templates/form.php

```diff
@@ -61,9 +61,9 @@
                     <!-- Topic segment --> 
                     <div id="customer-feedback-topics" class="customer-feedback-topics feedback-answer-no u-margin__top--4" style="display:none">
 
-                        <label for="customer-feedback-comment-topic-<?php echo $num; ?>" class="c-typography c-typography__variant--h3">
+                        <legend class="c-typography c-typography__variant--h3">
```

Comment:
It seems like the specification that legends always should be combined with the <fieldset> tag.

Reply:
vilken tag? ;D

Reply:
Ouch. Github stripped the tag from the comment. I was talking about the fieldset tag.